### ISSUE TITLE
Why not use sentence-mask task in GLM-130B

### ISSUE THREAD

**Issue by mx8435** [THUDM/GLM-130B#74]
Hi, 
    Thanks for your great job. The GLM paper _GLM: General Language Model Pretraining with Autoregressive Blank Infilling_ has three pretraining tasks (span-mask, sentence-mask, doc-level-mask). Why GLM-130B only use span-mask and doc-level-mask tasks but discard sentence-mask task ? Thanks.

----------
UPDATE.
I've read the [issue](https://github.com/THUDM/GLM-130B/issues/12) about missing [sMASK], can you share the detail about `since it harms the efficiency of pre-training`. It means performance degradation or train speed become slow ? Any numbers provided will be good. Thanks.

**Comment by Sengxian** [comment]
Hello @mx8435, this has nothing to do with training efficiency, we just remove [sMASK] for simplicity because we didn't come up with a unified solution to split paragraphs into sentences for Chinese, English and code data.
